scribbles/apps/authentication/forms.py

```diff
@@ -0,0 +1,13 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+
+
+class UserRegisterForm(UserCreationForm):
+    email = forms.EmailField(required=True, label="Email Address", initial="Your email address", error_messages={'required': 'This field is required'})
+    first_name = forms.CharField(required=True, initial="Your first name", label="First Name", error_messages={'required': 'This field is required'})
+    last_name = forms.CharField(required=True, initial="Your lastname", label="First Name", error_messages={'required': 'This field is required'})
+
+    class Meta:
+        model = User
+        fields = ['username', 'first_name', 'last_name', 'email', 'password1', 'password2']
```

Comment:
Since we're dealing with building an API, we should be using models.py instead of forms.py to represent our user table in the DB.